### PR TITLE
[minor] Add new Manage bundle server config for SNO

### PR DESCRIPTION
**Pull request by andrercm**
This PR adds the ability to auto-detect and override the storage classes for Manage PVCs (used by Doclinks, BIM and JMS configurations)

In cli mas install, if you choose to configure Manage with JMS, you will be prompted to confirm the storage class to be used.

Also, this adds a new Manage server bundle config type inherited from ansible-devops called `snojms` which enables the JMS queue in a smaller footprint, which is helpful for SNO type of environments. Therefore, in cli, now if JMS is enabled, then this new type `snojms` will be automatically set in case the install is detected to be into an SNO.

image

that will be passed along to the cli pipeline:

image

**This PR needs to go along with ansible-devops change:**
https://github.com/ibm-mas/ansible-devops/pull/841

### CODE REVIEW

**Review by durera — APPROVED** [review]

File: image/cli/mascli/functions/pipeline_config_manage_storage_classes [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+function config_pipeline_manage_storage_classes() {
+  echo ""
+  echo "Trying to auto-detect available storage classes for Manage application..."
+  echo ""
+  reset_colors
+  # 1. ROKS
+  oc get storageclass ibmc-file-gold-gid &>> $LOGFILE
+  if [[ $? == "0" ]]; then
+    MANAGE_STORAGE_CLASS_PROVIDER=ibmc
+    MANAGE_STORAGE_CLASS_RWX=ibmc-file-gold-gid
```
Comment (durera) [inline image/cli/mascli/functions/pipeline_config_manage_storage_classes]: Why does manage need it's own RWX/RWO storage class defined?

**Review by durera — DISMISSED** [review]

File: image/cli/mascli/functions/pipeline_config_manage_storage_classes (hunk 1 above)
Comment (andrercm) [reply image/cli/mascli/functions/pipeline_config_manage_storage_classes]: made the required changes to reuse the general RWX or RWO (if it's SNO) storage classes defined.

File: docs/changes.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ## Changes
 
-
+- [`5.4`](https://github.com/ibm-mas/cli/releases/tag/5.4.0) Add new Manage bundle server config for SNO
```
Comment (whitfiea) [inline docs/changes.md]: It would still be 5.3 I believe as 5.2 is the lastest at the moment

**Review by whitfiea — CHANGES_REQUESTED** [review]

File: docs/changes.md (hunk 2 above)
Comment (andrercm) [reply docs/changes.md]: fixed

**Review by whitfiea — APPROVED** [review]

**Pull request merged**